File: Lab_9/9/Storage.py
class Storage:
    def __init__(self, standart_photoNumber = 0, standart_maxPhotoNumber = 100000):
        self.__photoNumber = standart_photoNumber
        self.__maxPhotoNumber = standart_maxPhotoNumber
    def deletePhoto(self):
        if self.__photoNumber == 0:
            print("Error. Storage is empty")
        else:
            self.__photoNumber = self.__photoNumber - 1
            
    def makePhoto(self):
        if self.__photoNumber == self.__maxPhotoNumber:
            print("Error. Storage is full")
        else:
            self.__photoNumber = self.__photoNumber + 1
            
    def getPhotoNumber (self):
        return self.__photoNumber
    
    def getMaxPhotoNumber (self):
        return self.__maxPhotoNumber
        

File: Lab_2/src/ki304/Kisil/Lab2/Storage.java
package ki304.Kisil.Lab2;
/**
 * <code>Storage</code> implement storage of camera
 */
public class Storage
{
int photoNumber;
int maxNumber;

/**
 * Constructor
 */
public Storage()
{
photoNumber = 0;
maxNumber = 1000;
}

/**
 * @param <code>num</code> numbers of photos
 */
public Storage(int num)
{
photoNumber = num;	
maxNumber = 10000;
}
/**
 * @param <code>num</code> numbers of photos
 * @param<code>max</code> max numbers of photo
 */
public Storage(int num, int max)
{
photoNumber = num;	
maxNumber = max;
}

/**
 * Method simulate deleting photo
 */
public boolean deletePhoto()
{
	if(photoNumber>0) {
photoNumber--;	
return true;}
	else
	{System.out.print("Storage Error"+"\n");
	return false;}
}
/**
 * Method simulate making photo
 */
public boolean makePhoto()
{if(photoNumber<maxNumber)
{
photoNumber++;	
return true;
}
else
{
	System.out.print("Storage Error"+"\n");
	return false;
}

}
/**
 * Method returns numbers of photos in storage
 * @return current number of photos
 */
public int getPhotoNumber()
{
	return photoNumber;
}

/**
 * Method returns numbers of photos in storage
 * @return current number of photos
 */
public int getPhotoNumberLeft()
{
	return maxNumber-photoNumber;
}

}